examples/ucnf-kiknos/Makefile

```diff
@@ -0,0 +1,161 @@
+# Parent image for the NSE
+VPP_AGENT = ciscolabs/kiknos:latest
```

Comment:
Looks like this setting is causing the vL3 build to fail.  Since the kiknos NSE build is leveraging the generated make target stuff from `targets.mk / docker-targets.mk` for its docker build step there's not a great way to set the VPP_AGENT value only for Kiknos builds.  

Without doing a rework of the kiknos Makefile, I think you could do one of: 
1.  Add a conditional at the examples/Makefile level for setting VPP_AGENT to the Kiknos value if a setting is passed.
2.  Change your kiknos build instructions to indicate that the VPP_AGENT setting always needs to be set to the `ciscolabs/kiknos:...` value for kiknos builds.  
3.  Change the vL3 NSE Makefile to use the specific VPP_AGENT value instead of using the var it uses now from the global makefile.

I think I'm leaning toward option 3 for now and we can add a follow-on issue for figuring out the best Makefile target organization.  I'm not a fan of the go build in the dockerfile approach due to difficulties in the dev workflow and CI.

Reply:
Well maybe I got the vL3 build fail reason wrong since you already changed it like item 3.  Looking in more detail at the build failure.

Reply:
yeah, it's due to this line and the fact that the vL3 build is using the same VPP_AGENT var:
```
Step 2/6 : FROM ${VPP_AGENT} as runtime
 
latest: Pulling from ciscolabs/kiknos
 ```

Reply:
I already fixed this

Reply:
It's not fixed... see my comment on the vl3 Makefile change for the fix to the fix.